Modbus.Ports/ModbusClientR444A01.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Linq;
+using EasyModbus;
+
+
+namespace Modbus.Ports
+{
+    /// <summary>
+    /// Modbus Client for the Sensor R444A01
+    /// e.g. https://www.amazon.de/Temperatur-Feuchtigkeitssensor-Ziffernanzeige-Feuchtigkeit-Temperatursensor-Externem/dp/B078NR8DDJ
+    /// e.g. https://www.aliexpress.com/item/33054683552.html
+    /// </summary>
+    public class ModbusClientR444A01
+    {
+        private const int TemperatureAddressRegister = 0x00;
+        private const int HumidityAddressRegister = 0x01;
+        private const int SlaveAddressRegister = 0x02;
+        private const int BaudRateAddressRegister = 0x03;
+        
+        private readonly ModbusClient _ModbusClient;
+        private int _slaveAddress;
+
+        public ModbusClientR444A01(string comPort)
+        {
+            comPort = string.IsNullOrWhiteSpace(comPort) ? throw new ArgumentNullException(nameof(comPort)) : comPort;
+            _ModbusClient = new ModbusClient(comPort);
+        }
+
+        public ModbusClientR444A01(string ipAddress, int port)
+        {
+            throw  new NotSupportedException("Not supported yet");
+        }
+
+        private void SetupModbusClient()
+        {
+            //modbusClient.UnitIdentifier = 1; //Not necessary since default slaveID = 1;
+            _ModbusClient.Baudrate = 9600;	// Not necessary since default baudrate = 9600

```

Comment:
add method Parameter to set Baudrate different in case the user changged the bautrade on the sensor